savant/entrypoint/main.py

```diff
@@ -36,6 +38,21 @@ def main(config_file_path: str):
     update_logging(config.parameters['log_level'])
     logger = get_logger('main')
 
+    if status_filepath is not None:
+        healthcheck_port = config.parameters.get('healthcheck_port')
+        if healthcheck_port:
+            healthcheck_server = HealthCheckHttpServer(
+                host='',
+                port=healthcheck_port,
+                http_path='/healthcheck',
+                status_filepath=status_filepath,
+            )
+            healthcheck_thread = Thread(
+                target=healthcheck_server.serve_forever,
+                daemon=True,
+            )
+            healthcheck_thread.start()
```

Comment:
Wouldn't there be some problem with the thread not stopping when the pipelines are terminated?

Reply:
No, this is a daemon thread.